src/MatchMaker/Encounter/Encounter.php

```diff
@@ -19,6 +19,8 @@ class Encounter
 {
     public const STATUS_PLAYING = 'En cours';
     public const STATUS_OVER = 'Terminé';
+    public int $scorePlayerA;
+    public int $scorePlayerB;
```

Comment:
Good idea !
To encourage using the methods, I suggest them to be private :)
Also, they have to be set with a default value. Because we are checking over null, they should be nullable with null by default.

Reply:
That's right indeed! Good idea with the private ones :)